### PR TITLE
Fix doc, to build a minimal plugin

### PR DESCRIPTION
**Pull request by Xx-Parks-xX**

### CODE REVIEW

**Comment by clarsonneur** [comment]
@Xx-Parks-xX, I think you need to rebase and re-submit, so I can merge it.

File: README.md [hunk 1]
```diff
@@ -93,21 +93,43 @@ This code will implement the FORJJ plugin protocol.
...
+    ```
+
     ```bash
     go generate           # To generate the flags management code.
     ```
-  Depending on your plugin definition (`/runtime/service_type`), `go generate` will create several files:
+    Depending on your plugin definition (`/runtime/service_type`), `go generate` will create several files:
+
+    ```bash
+    glide update           # Update dependencies
```
Comment (uggla) [inline README.md]: Need to clear the glide cache with `glide cc` before to ensure the latest packages have been downloaded.

Comment (uggla) [reply README.md]: Note : this need to be done before the `glide get` command. However this is not useful if it is the first time doing this.

File: README.md [hunk 2]
```diff
@@ -93,21 +93,43 @@ This code will implement the FORJJ plugin protocol.
...
+    ```bash
+    glide init           # Initialize of glide.yaml and glide.lock
+    ```
+
+    ```bash
+    glide get github.com/forj-oss/goforjj         # Get the goforjj project in te tree to access the template
+    ```
+
     ```bash
     go generate           # To generate the flags management code.
```
Comment (uggla) [inline README.md]: Could be interesting to add a `-v` to the command, so it shows the files that are generated.

Comment (clarsonneur) [reply README.md]: go generate show everytime what is generated. only files generated.

File: README.md (hunk 1 above)
Comment (clarsonneur) [reply README.md]: ok. So, you need to fix the documentation, right? I will wait for the fix then.

Comment (uggla) [reply README.md]: Yep I expect @Xx-Parks-xX to update it accordingly on Monday.

Comment (uggla) [reply README.md]: Yep I expect @Xx-Parks-xX to update it accordingly on Monday.

**Pull request merged**